src/spikeinterface/core/waveform_extractor.py

```diff
@@ -556,7 +556,11 @@ def is_extension(self, extension_name) -> bool:
                     and "params" in self._waveforms_root[extension_name].attrs.keys()
                 )
 
-    def load_extension(self, extension_name):
+    def is_extension(self, extension_name) -> bool:
+        warn("WaveformExtractor.is_extension is deprecated! Use `has_extension` instead.")
```

Comment:
We should say exactly when it will be deprecated.

Reply:
Well it's deprecated since now, but will be removed in version xx
I thought I would leave the choice of which version to you guys :)

Reply:
```suggestion
        warn("WaveformExtractor.is_extension is deprecated and will be removed in version 0.102.0! Use `has_extension` instead.")
```
Since this will make it into v100: 2 down the line would be 102 if @h-mayorquin wants two versions.  (2 seems fair although I could also see an argument for 1 depending on release speeds). 

I agree overall that the code base does have some deprecation warnings without explicit removal versions which makes it hard to clean up some of the legacy code so I think moving forward having a specific version number would be nice. @alejoe91 do you have a desired number of versions for deprecations like this just so everyone can be on the same page.

Reply:
Let's do two versions from the current one then?

Reply:
Thanks, this is fine for me.